pom.xml

```diff
@@ -268,7 +268,7 @@
     <version.org.springframework.boot>2.7.18</version.org.springframework.boot>
     <version.org.springframework.osgi>1.2.1</version.org.springframework.osgi>
     <!-- spring-security.version and tomcat.version overrides version from spring-boot-dependencies bom -->
-    <spring-security.version>5.7.12</spring-security.version>
+    <spring-security.version>5.7.11</spring-security.version>
```

Comment:
```suggestion
```
@yesamer Thanks for the PR. `spring-boot-dependencies` [BOM](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.7.18/spring-boot-dependencies-2.7.18.pom) has 5.7.11 default. So removing from here it is also good.

Reply:
@akumar074 Great!! 
According to this message:
`    <!-- spring-security.version and tomcat.version overrides version from spring-boot-dependencies bom -->
`
Can we do the same for tomcat? WDYT?

Reply:
Nevermind, I noticed tomcat is defined here https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2510/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R156 as well, so I guess this is required to keep the versions synchronized.